commands/user_e2e_test.go

```diff
@@ -136,3 +138,67 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestResetUserMfaCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	user, appErr := s.th.App.CreateUser(&model.User{Email: s.th.GenerateTestEmail(), Username: model.NewId(), Password: model.NewId(), MfaActive: true, MfaSecret: "secret"})
+	s.Require().Nil(appErr)
+
+	s.RunForSystemAdminAndLocal("Reset user mfa", func(c client.Client) {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableMultifactorAuthentication
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableMultifactorAuthentication = true })
+		defer func() {
+			s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableMultifactorAuthentication = *previousVal })
+		}()
```

Comment:
nit: the wrapping `func` is not needed, same in the rest of the `defer` oneliners

```suggestion
		defer s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableMultifactorAuthentication = *previousVal })
```